conference/models.py

```diff
@@ -813,6 +813,15 @@ def getAbstract(self, language=None):
             self, "abstracts", language
         )
 
+    def get_abstract(self, language=None):
```

Comment:
there is a getAbstract function  above it, so maybe we could either use it here, or mark the one above as deprecated.

Reply:
The problem with `getAbstract` is that to get the abstract contents you need to call `getAbstract().body`; however unlikely, if there is no abstract, this will lead to issues in the templates. And I don't want to modify the existing `getAbstract` as it's used in many places - I'll leave a TODO that these two should be merged.